Lessen_5_hw/task_3.js

```diff
@@ -0,0 +1,72 @@
+/*
+Создать программу, которая будет принимать на вход СЛОВО (создать переменную со словом),и выводить в консоль количество гласных и согласных букв в этом слове. 
+Ответ должен выводиться шаблонным литералом вида word contains x vowels and y consonants
+*/
+const vowelList = 'aeiouy'
+const consonantsList = 'bcdfghjklmnpqrstvwxz'
+let vowelCount = 0
+let consonantsCount = 0
+let str = 'I am super engineer'
+str = str.toLowerCase()
+for (let i = 0; i < str.length; i++) {
+	if (vowelList.indexOf(str[i]) !== -1) {
+		vowelCount += 1
+	} else if (consonantsList.indexOf(str[i]) !== -1) {
+		consonantsCount += 1
+	}
+}
+console.log(
+	`word contains vowels ${vowelCount} and ${consonantsCount} consonants`
+)
+console.log()
+/*
+Написать программу, которая видоизменяет принимаемое слово (переменная со словом) шифром ЦЕЗАРЯ (посмотреть в википедии) со сдвигом на 1 в любую из сторон. 
+Направление шифрования задается переменной offset, которая может быть +1 и -1.
+*/
+const alphabet = [
+	'a',
+	'b',
+	'c',
+	'd',
+	'e',
+	'f',
+	'g',
+	'h',
+	'i',
+	'j',
+	'k',
+	'l',
+	'm',
+	'n',
+	'o',
+	'p',
+	'q',
+	'r',
+	's',
+	't',
+	'u',
+	'v',
+	'w',
+	'x',
+	'y',
+	'z',
+]
+const input = 'AaA'
```

Comment:
Try to modify solution to handle such case `Aa 1A` - you need to keep a space and a number in the result string